### DISCUSSION TITLE
"Mark as learned" in VS Code extension

### DISCUSSION THREAD

**Discussion by curiousYi** [mattpocock/ts-error-translator#168] [Q&A] [answered]
First off, I love the tool in terms of improving dx and for learning.

My question:
Is there a way to reset your "Mark as Learned" history or is it permanent for now?

Similar to this issue.
https://github.com/mattpocock/ts-error-translator/issues/166

**Comment by mattpocock** [comment, marked as answer]
Go into your VSCode settings (as JSON) and reset them manually!

**Comment by curiousYi** [comment]
🙌 Thanks! Adding more details below in case it helps other people.

Steps:
1. Go to user settings json (could be in workspace settings etc)
2. Look for `totalTypescript.hiddenTips` array. 
3. Delete tips.
